tests/test_predict.py

```diff
@@ -69,3 +72,48 @@ def test_predict(model_class, env_id, device):
 
         action, _ = model.predict(vec_env_obs, deterministic=False)
         assert action.shape[0] == vec_env_obs.shape[0]
+
+
+class FlattenBatchNormExtractor(BaseFeaturesExtractor):
+    """
+    Feature extract that flatten the input and uses batch normalization.
+    Used as a placeholder when feature extraction is not needed.
+
+    :param observation_space:
+    """
+
+    def __init__(self, observation_space: gym.Space, batch_norm_elements=4):
+        super(FlattenBatchNormExtractor, self).__init__(observation_space, get_flattened_obs_dim(observation_space))
+        self.flatten = nn.Flatten()
+        self.batch_norm = nn.BatchNorm1d(batch_norm_elements)
```

Comment:
you should probably also add a dropout layer and check that calling `predict` with `deterministic=True` is actually deterministic

Reply:
Done. See commit 5425159fb78101c5946f705df253ebcaf6a373c9